drop-transfer/src/ws/client/v2.rs

```diff
@@ -265,6 +265,20 @@ impl<const PING: bool> handler::HandlerLoop for HandlerLoop<'_, PING> {
         futures::future::join_all(tasks).await;
     }
 
+    async fn on_conn_break(&mut self) {
+        debug!(self.logger, "Waiting for background jobs to pause");
+
+        let tasks = self.tasks.drain().map(|(_, task)| {
+            task.job.abort();
+
+            async move {
+                task.events.pause().await;
+            }
+        });
+
+        futures::future::join_all(tasks).await;
+    }
+
```

Comment:
Would it make sense to move these identical instances to a common place?

Reply:
Yes, I would be delighted to do it. But that means a refactoring since the task struct is not unified yet